CorsixTH/Lua/entities/humanoids/patient.lua

```diff
@@ -1027,7 +1027,7 @@ function Patient:updateMessage(choice)
         -- no requirements missing
         enabled = false
       end
-    else -- if choice == "guess_cure" then
+    --else -- if choice == "guess_cure" then
```

Comment:
drop the -- after else if you're putting it before else. Also space.

We don't need both this and the TODO, so maybe drop this entirely and change the TODO to `TODO: implement choice == "guess_cure"`